sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -54,7 +53,7 @@ import {
   formatFieldsForSelect
 } from "./internal/helpers";
 import { tracingPolicy } from "@azure/core-http";
-import { Spanner } from "./internal/tracingHelpers";
+import { trace as traceFromTracingHelpers } from "./internal/tracingHelpers";
```

Comment:
why the clunky alias?

Reply:
To make it obvious if I accidentally bypassed my internal method (that I use so I can stub it out for testing later)